distribution/docker/wait.sh

```diff
@@ -24,7 +26,7 @@ while [ ${cmd} -ne 1 ]
 do
     >&2 echo "DDF is NOT up - sleeping for 10 seconds"
     sleep 10s
-    cmd=$(curl -s -k -i -X GET "https://${_sut_host}:${_sut_port}/admin/jolokia/exec/org.apache.karaf:type=feature,name=root/infoFeature(java.lang.String)/profile-${_target_profile}" | grep -i '"Installed":true' | wc -l)
+    cmd=$(curl -s -k -i -X GET "https://${_sut_host}:${_sut_port}/admin/jolokia/exec/org.apache.karaf:type=feature,name=root/infoFeature(java.lang.String)/${_target_feature}" | grep -i '"Installed":true' | wc -l)
```

Comment:
❓ Since the commands outside and inside the do loop are identical, could we save them off to be exec'd instead of having to maintain two lines in the script?

Reply:
I tried to do that last time and that wasn't working very well. I can try again